src/_utils/_git_secret_tools.sh

```diff
@@ -147,6 +147,12 @@ function _os_based {
 
 # File System:
 
+function _clean_path {
+  # This function transforms windows paths to *nix paths
```

Comment:
Can you show an example of a path transformation in a comment here? For example, 

`# this code would transform c:\this\that.file (or whatever) to XYZ`

Reply:
Also, how about we rename this function to `_clean_windows_path`, 
since it only effects paths with drive letters (which are a windows thing).
 
If you have a better idea for a function name that works too.